smithy-aws-traits/src/test/resources/software/amazon/smithy/aws/traits/diffs/sigv4-sigv4a-swap-order.events

```diff
@@ -0,0 +1,31 @@
+[DANGER] ns.foo#Service1: Changed trait `smithy.api#auth` from 
```

Comment:
Since the purpose of this test it to test for behavior of Sigv4Migration events, could add suppression for these events here, like in the tests above. It decouples from the behavior of breakingChange too, e.g., if you change the message in smithy-model, you don't have to update these tests in smithy-aws-traits.

Moreover, why are there no migration events here for service1, where sigv4a is being moved ahead of sigv4. Isn't the risk similar to the `sigv4a-before-sigv4` case?